packer/nomad-server/scripts/download-files.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash -x
+
```

Comment:
Documenting how this relies on the `urls` file, and its specific format would be nice.